File: frontend/src/types/shift.ts
export type Priority = "info" | "warning" | "urgent";

export interface ShiftSummary {
  id: string;
  tenantId?: string;
  title: string;
  body: string;
  priority?: Priority;
  author: string;
  createdAt: string;
}

export interface Shift {
  id: string;
  tenantId: string;
  staffMemberId: string;
  staffName: string;
  startTime: string;
  endTime: string;
  notes?: string;
}

File: backend/src/DineOS.Domain/Entities/Shift.cs
using DineOS.Domain.Common;

namespace DineOS.Domain.Entities;

public class Shift : TenantAuditingEntity
{
    public long StaffMemberId { get; set; }
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }
    public string? Notes { get; set; }

    public StaffMember? StaffMember { get; set; }
}
